application/db/ethereumTxToKafka.go

```diff
@@ -30,7 +32,7 @@ func (t *EthereumTxToKafka) Value() ([]byte, error) {
 }
 
 func (t *EthereumTxToKafka) Validate() error {
-	if t.TxHash.String() == "0x0000000000000000000000000000000000000000000000000000000000000000" {
+	if t.TxHash.String() == constants.EthereumEmptyTxHash {
```

Comment:
Why don't we `if t.TxHash == common.Hash" ? I believe we don't need `.String()` extra step to compare arrays.